src/components/Chart/Maps.js

```diff
@@ -189,6 +221,56 @@ export class GenericMap extends Component {
       })
     })
   }
+  renderPointTooltip() {
+    const {
+      hoverPoint,
+      layout: {
+        projectPoint,
+        numberFormat
+      }
+    } = this.state
+    const {
+      width,
+      pointLabel,
+      pointTooltips,
+      unit,
+    } = this.props
+    if (!hoverPoint) {
+      return null
+    }
+    const [ x, y ] = projectPoint([hoverPoint.datum.lon, hoverPoint.datum.lat])
+
+    const value = isNaN(hoverPoint.datum.value) 
+      ? new String(hoverPoint.datum.value).trim()
+      : numberFormat(hoverPoint.datum.value)
+
+    const body = pointTooltips.map(t => {
+      const val = hoverPoint.datum[t]
+      if (val) {
+        return (<>{`${t}: ${ isNaN(val) ? val : numberFormat(+val) }`}<br/></>)
```

Comment:
I like the neatness of using the column name as a label / unit.

However the current implementation limits everything to this style:
> **New York**
> 1784 km<sup>2</sup>
> Einwohner: 862'300

But normally we use the unit as a suffix:
> **New York**
> 1784 km<sup>2</sup>
> 862'300 Einwohner

Also we might have different number formatting requirements for the different values. E.g.
> **New York Area**
> 17,1 Tonnen CO<sub>2</sub> pro Einwohner
> 13,6 Mio. Einwohner

First line would be `.1f` and second `.3s`. I don't think it worth abstracting all those problems and would just expect pre formatted details lines, in this case `13,6 Mio. Einwohner` – no type detecting, casting or `numberFormat` here. But do pipe those detail lines through `subsup` to support fancy stuff like CO<sub>2eq</sub>.